libpromises/evalfunction.c

```diff
@@ -238,8 +238,8 @@ static JsonElement* VarRefValueToJson(EvalContext *ctx, const FnCall *fp, const
             break;
 
         case RVAL_TYPE_CONTAINER:
-            convert = value;
-            *allocated = false;
+            convert = JsonCopy(value);
```

Comment:
Thanks for reviewing @tzz, I was ready to ping you. :-)

So this line always spewed a compiler warning:
```
evalfunction.c:241:21: warning: assignment discards 'const' qualifier from pointer target type [-Wdiscarded-qualifiers]
             convert = value;
                     ^
```